helm/cluster-api/files/core/bases/clusterclasses.cluster.x-k8s.io.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     app.kubernetes.io/version: '{{ .Chart.AppVersion }}'
     cluster.x-k8s.io/provider: cluster-api
     helm.sh/chart: cluster-api
-  name: machinepools.exp.cluster.x-k8s.io
+  name: clusterclasses.cluster.x-k8s.io
```

Comment:
ClusterClass CRD got added, and exp MachinePool got removed, so it shows it here as renaming.